.envrc

```diff
@@ -0,0 +1,2 @@
+watch_file nix/shell.nix
+use flake
```

Comment:
Do we have to have this? I personally use direnv/Nix in other projects, but not in this one and it's a bit weird that I now have to disable this project, when we could turn it around and `gitignore` the `.envrc` so everyone can decide on their own.